### DISCUSSION TITLE
NudeDetector

### DISCUSSION THREAD

**Discussion by 0xfade666** [DavidRomanovizc/DatingBot#113] [Q&A] [answered]
line 15, in <module>
    from nudenet import NudeDetector
ImportError: cannot import name 'NudeDetector' from 'nudenet' (/usr/local/lib/python3.10/dist-packages/nudenet/__init__.py)

**Comment by dzhalaevd** [comment, marked as answer]
> line 15, in from nudenet import NudeDetector ImportError: cannot import name 'NudeDetector' from 'nudenet' (/usr/local/lib/python3.10/dist-packages/nudenet/**init**.py)

Check the library version. It should be NudeNet==3.0.8

**Comment by 0xfade666** [comment]
thx
